velox/common/memory/tests/MemoryAllocatorTest.cpp

```diff
@@ -1194,6 +1194,14 @@ TEST_P(MemoryAllocatorTest, allocation) {
   allocation->clear();
 }
 
+TEST_P(MemoryAllocatorTest, exceedLargestSizeClass) {
+  const size_t kExceedLargestSizeClass = instance_->largestSizeClass() + 1;
```

Comment:
Can you test with different allocation size based on the class size and check the number of page runs in the resulting allocation? 
```
AllocationPool::allocationAt(0)?
1, => expect one page run?
first size class + 1 => expect two page runs?
second size class + 1 => expect two page runs?
...
largestSizeClass() + 1 => expect two page runs?
```
You can structure the test with a while loop and each iteration test different size + expected page run? Thanks!

Reply:
Thanks for your comments! I tried to add these checks but I found in most cases `allocation->numRuns() == 1`. Is it as expected, or did I misunderstand the above logic? Please review the updated code.

Reply:
I think the number of page runs is most likely 1. I previous comment on the expected page run is incorrect. But you need to update allocation object as newRun() will insert new run: AllocationPool::allocationAt(currentRunIndex()). Then it suppose all the newRun should have only one pageRun.